application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -92,12 +95,20 @@ public void onMessageContext(MessageContextInteractionEvent event) {
         String originalChannelId = event.getTarget().getChannel().getId();
         String authorId = event.getTarget().getAuthor().getId();
         String mostCommonTag = tags.get(0);
+        String chatGptPrompt = "Generate title for question:"
+                                + originalMessage + ". Minimum " + TITLE_MIN_LENGTH
+                                + " and maximum " + TITLE_MAX_LENGTH +" characters. Return only the title.";
+        Optional<String> chatGptResponse = chatGptService.ask(chatGptPrompt, "");
```

Comment:
do you have access to thread tag here? might be a good idea to use it as context instead of empty string